dev/DeploymentAgent/pch.h

```diff
@@ -0,0 +1,13 @@
+﻿#pragma once
```

Comment:
Missing `// Copyright...`

same comment applies throughout

Reply:
Yeah.. Copy paste from RestartAgent. I notice a lot of WinAppSDK files (ex: all of PushNotifications) aren't having Microsoft Copyright in the file header.